.gitignore

```diff
@@ -334,3 +334,6 @@ inspectcode
 
 # BenchmarkDotNet
 /BenchmarkDotNet.Artifacts
+
+# Discord Plugin Integration
+.idea/.idea.osu.Desktop/.idea/discord.xml
```

Comment:
I think what @peppy intended is to edit your *global `.gitignore` on your actual local machine*, not this repository's. Git can have per-machine ignores that span all repositories.

Reply:
Uh, i've never tried this, lemme check that